app/helpers/layout_helper.rb

```diff
@@ -81,7 +81,7 @@ def page_entries_info(collection, options = {})
   end
 
   def will_paginate_with_info(collection = nil, options = {})
-    content_tag(:div, :id => "pagination", :class => "row") do
+    content_tag(:div, :id => "pagination", :class => "row", 'data-meta-count' => collection.total_entries) do
```

Comment:
I've added this particularly  to pagination div, so the total collection size would be collectable from any pagination page.

As you can see at the following screenshot, the total size is visable though hardly collectable. 
![total](https://cloud.githubusercontent.com/assets/11807069/23836664/6dffb8d8-0784-11e7-924f-806e385fbb1b.png)
 The meta name `per page` exists only in hosts page.